src/ops.js

```diff
@@ -63,6 +63,21 @@ window.splinterlands.ops = (function() {
 		});
 	}
 
+	async function cancel_cards_rental(market_item_id) {
+		return splinterlands.send_tx_wrapper('market_cancel_rental', 'Cancel Rental', { items: [market_item_id] }, async tx => {
+			if(tx && tx.trx_info && tx.trx_info.result) {
+				let result = JSON.parse(tx.trx_info.result);
+
+				if(result && result.market_items && Array.isArray(result.market_items) && result.market_items.length > 0 && result.market_items[0].next_rental_payment) {
+					console.log(result);
+					console.log(result.market_items[0].next_rental_payment);
+				}
+			}
```

Comment:
Can this code can be deleted? It doesn't look like it's doing anything.

Reply:
For the lines 68 to 75, I agree it can be removed, but the main function is necessary to cancel the rental. I could not find any other function to cancel the cards rental. Is there any?

Reply:
I was just talking about lines 68 to 75. Remove that code and I think it should be good.

Reply:
Done